app/code/Magento/QuoteGraphQl/Model/Resolver/ShippingAddress/SelectedShippingMethod.php

```diff
@@ -31,36 +31,37 @@ public function resolve(Field $field, $context, ResolveInfo $info, array $value
         /** @var Address $address */
         $address = $value['model'];
         $rates = $address->getAllShippingRates();
-        $carrierTitle = null;
-        $methodTitle = null;
+        $carrierTitle = '';
```

Comment:
Please remove redundant lines.

Reply:
> Please remove redundant lines.
In case I'll remove these lines the $carrierTitle and the $methodTitle variables might not be defined (please check lines 46, 47).
Should I add any changes here?
Thank you.